### PR TITLE
Add saved query cmdlets

### PR DESCRIPTION
**Pull request by dolauli**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/ResourceGraph/ResourceGraph/ChangeLog.md [hunk 1]
```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 
+## Version 0.7.8
```
Comment (dingmeng-xue) [inline src/ResourceGraph/ResourceGraph/ChangeLog.md:22]: Expected version should be 0.8.0

File: src/ResourceGraph/ResourceGraph/help/Az.ResourceGraph.md [hunk 2]
```diff
@@ -1,16 +1,25 @@
...
 Download Help Link: https://docs.microsoft.com/en-us/powershell/module/az.resourcegraph
-Help Version: 0.1.0.0
+Help Version: 1.0.0.0
 Locale: en-US
 ---
 
 # Az.ResourceGraph Module
 ## Description
-This topic displays help for the Azure Resource Graph cmdlets.
+Microsoft Azure PowerShell: ResourceGraph cmdlets
```
Comment (dingmeng-xue) [inline src/ResourceGraph/ResourceGraph/help/Az.ResourceGraph.md:11]: Search-AzGraph is deleted?

File: src/ResourceGraph/ResourceGraph.sln [hunk 3]
```diff
@@ -21,6 +21,10 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ScenarioTest.ResourceManage
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TestFx", "..\..\tools\TestFx\TestFx.csproj", "{BC80A1D0-FFA4-43D9-AA74-799F5CB54B58}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Authenticators", "..\Accounts\Authenticators\Authenticators.csproj", "{E8880B70-568E-48C5-8CB9-D577BC958FB2}"
+EndProject
```
Comment (erich-wang) [inline src/ResourceGraph/ResourceGraph.sln:25]: Please remove duplicate project Authenticators

File: src/ResourceGraph/ResourceGraph/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -25,5 +25,5 @@
 [assembly: CLSCompliant(false)]
 [assembly: Guid("cdfd4e1b-bb5f-439b-8843-e6127cd9b8ad")]
 
-[assembly: AssemblyVersion("0.7.7")]
-[assembly: AssemblyFileVersion("0.7.7")]
+[assembly: AssemblyVersion("0.7.8")]
```
Comment (dingmeng-xue) [inline src/ResourceGraph/ResourceGraph/Properties/AssemblyInfo.cs:28]: 0.8.0

File: src/ResourceGraph/ResourceGraph/help/Az.ResourceGraph.md [hunk 5]
```diff
@@ -1,16 +1,28 @@
-﻿---
+---
 Module Name: Az.ResourceGraph
-Module Guid: a290eead-f293-4588-b61e-b465b237591e
+Module Guid: 2c8440b2-9a3f-4b55-b024-d88cf74f7bd7
```
Comment (dingmeng-xue) [inline src/ResourceGraph/ResourceGraph/help/Az.ResourceGraph.md:3]: Can you use original module ID?

File: src/ResourceGraph/ResourceGraph/ChangeLog.md [hunk 6]
```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 
+## Version 0.8.0
+* Added four new cmdlets -- Get-AzResourceGraphQuery, New-AzResourceGraphQuery, Remove-AzResourceGraphQuery, Update-AzResourceGraphQuery
```
Comment (dingmeng-xue) [inline src/ResourceGraph/ResourceGraph/ChangeLog.md:23]: "Added new cmdlets to support query resource"

File: src/ResourceGraph/ResourceGraph.sln [hunk 7]
```diff
@@ -1,7 +1,7 @@
 
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.28307.136
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29613.14
```
Comment (dingmeng-xue) [inline src/ResourceGraph/ResourceGraph.sln:4]: Can we keep original version?

File: src/ResourceGraph/ResourceGraph/Az.ResourceGraph.psd1 [hunk 8]
```diff
@@ -65,13 +64,13 @@ RequiredAssemblies = 'Microsoft.Azure.Management.ResourceGraph.dll'
 # TypesToProcess = @()
 
 # Format files (.ps1xml) to be loaded when importing this module
-FormatsToProcess = 'ResourceGraph.format.ps1xml'
+FormatsToProcess = 'ResourceGraph.format.ps1xml', 'ResourceGraph.Autorest\Az.ResourceGraph.format.ps1xml'
 
 # Modules to import as nested modules of the module specified in RootModule/ModuleToProcess
-NestedModules = @('Microsoft.Azure.PowerShell.Cmdlets.ResourceGraph.dll')
+NestedModules = 'Az.ResourceGraph.psm1', 'ResourceGraph.Autorest\Az.ResourceGraph.psm1'
```
Comment (erich-wang) [inline src/ResourceGraph/ResourceGraph/Az.ResourceGraph.psd1:70]: [TODO] Az.ResourceGraph.psm1 should be added dynamically

**Review by erich-wang — APPROVED** [review]

**Comment by dingmeng-xue** [comment]
According to the quality of current CI env, suggest to postpone it.

**Review by dingmeng-xue — APPROVED** [review]

**Pull request merged**